ddl/ddl-schema.html.md.erb

```diff
@@ -90,6 +83,6 @@ The following system-level schemas exist in every database:
 
 -   `pg_catalog` contains the system catalog tables, built-in data types, functions, and operators. It is always part of the schema search path, even if it is not explicitly named in the search path.
 -   `information_schema` consists of a standardized set of views that contain information about the objects in the database. These views get system information from the system catalog tables in a standardized way.
--   `pg_toast` stores large objects such as records that exceed the page size. This schema is used internally by the HAWQ system.
+-   `pg_toast` stores large objects such as records that exceed the page size. HAWQ uses the EXTENDED strategy for column storage. It allows both compression and out-of-line storage. Compression will be attempted first, then out-of-line storage if the row is still too big.This schema is used internally by the HAWQ system.
```

Comment:
Missing a period at the end of the new sentence.  Also, is it confirmed that HAWQ uses the EXTENDED strategy? Last comment from Stanley was that this was a guess.